src/Bounds.cpp

```diff
@@ -733,6 +733,11 @@ class Bounds : public IRVisitor {
         }
     }
 
+    void visit(const Shuffle *op) {
+        // TODO: We could consider the bounds of the lanes used by the Shuffle.
```

Comment:
Actually, I'm mistaken about the need to vectorize 512 wide for that conversion. I thought it was a simpler example than what I first thought of, which is an interleaved RGB store, which needs 3x native vectors (384).